sdk/servicebus/service-bus/samples/javascript/gettingStarted/useProxy.js

```diff
@@ -0,0 +1,36 @@
+/*
+  This sample demonstrates how to create a ServiceBusClient meant to be used in an environment
+  where outgoing network requests have to go through a proxy server
```

Comment:
One thing I was thinking about here is you could also set up the proxy in this example, e.g. by adding the following:

```
httpProxy = require('http-proxy');
httpProxy.createProxyServer({target:'http://localhost:9000'}).listen(8000);
proxy.on('proxyReq', () => { console.log('proxying!') });
```

It would allow users to play with the actual proxy itself without having to install proxy software.